Sources/TuistKit/Commands/TestCommand.swift

```diff
@@ -161,11 +161,17 @@ public struct TestCommand: AsyncParsableCommand, HasTrackableParameters {
     var binaryCache: Bool = true
 
     @Flag(
-        help: "Run all tests instead of selectively test only those that have changed since the last successful test run.",
+        help: "Enable or disable selective testing feature. When selective testing is disabled, all tests will be run and no selective test results will be stored.",
         envKey: .testSelectiveTesting
     )
     var selectiveTesting: Bool = true
 
+    @Flag(
+        help: "Run all tests instead of selectively test only those that have changed since the last successful test run.",
+        envKey: .testRunAllTests
+    )
```

Comment:
Giving this one a look in-context trying to put the community proposal discussion aside confused me. Specially after reading the description, which reads as the opposite of what `--selective-testing`. I could be me, but I expect the combination of names to be somewhat confusing, so I lean on a deprecation nudge and a store-related flag to control the storing behaviour.

Reply:
I see `--no-selective-testing` as a way to completely disable the feature.
> Specially after reading the description, which reads as the opposite of what --selective-testing

It is the opposite, you are reading that right. I'm not sure if I follow the confusion? `--run-all-tests` is equivalent of `--no-selective-testing` for the phase of fetching local and remote selective testing results.